### DISCUSSION TITLE
Can't upgrade to 0.11 due to UWSGI issues; 0.9 works fine

### DISCUSSION THREAD

**Discussion by csolisr** [searx/searx#929] [Installation issue: custom] [answered]
I first installed Searx through the Yunohost installer available at https://github.com/abeudin/searx_ynh/ , but when I attempted to manually upgrade to 0.11 (since the bundled version is 0.9) I ran into issues when running the daemon through UWSGI. The Systemd log reads as follows:

```
systemd[1]: Starting LSB: Start/stop uWSGI server instance(s)...
uwsgi[22324]: Starting app server(s): uwsgi -> ! failed!
systemd[1]: uwsgi.service: control process exited, code=exited status=1
systemd[1]: Failed to start LSB: Start/stop uWSGI server instance(s).
systemd[1]: Unit uwsgi.service entered failed state.
```

I attempted configuring UWSGI to log, but since it doesn't even start the logging file is not generated at all.

**Comment by josch** [comment]
I am able to reproduce this.

It also failes on the Debian CI infrastructure since April with the same problem:

https://ci.debian.net/packages/s/searx/unstable/amd64/

I would've suspected that since uwsgi fails, this is due to a newer uwsgi version, but the version didn't change between the first failing run and the last successful run.

This has also been reported to Debian as https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=900068 and according to the bug reporter, fails in Ubuntu as well.

**Comment by josch** [comment, marked as answer]
@csolisr This is not yet a solution, merely a workaround (which might point to the solution) but does running the following fix your problem?

    sed -i 's/autoload = true/#autoload = true/' /usr/share/uwsgi/conf/default.ini

**Comment by utiltiy** [comment]
Hi,
if got the same problem. The workaround solved my problem and uwsgi starts now with my additional *.ini file.

Thanks!
